pkg/controller/ovn-ic.go

```diff
@@ -361,3 +365,75 @@ func genHostAddress(host string, port string) (hostaddress string) {
 	}
 	return hostaddress
 }
+
+func (c *Controller) SynRouteToPolicy() {
+
+	lr, err := c.ovnClient.GetLogicalRouter(util.DefaultVpc, false)
+	if err != nil {
+		klog.V(5).Infof("logical router does not exist %v at %v", err, time.Now())
+		return
+	}
+	lrRouteList, err := c.ovnClient.GetLogicalRouterRouteByOptions(util.OVNIC_Key, util.OVNIC_Value)
+	if err != nil {
+		klog.V(5).Infof("failed to list lr ovn-ic route %v", err)
+		return
+	}
+	if len(lrRouteList) == 0 {
+		klog.V(5).Info(" lr ovn-ic route does not exist")
+		lrPolicyList, err := c.ovnClient.GetLogicalRouterPoliciesByOptions(util.OVNIC_Key, util.OVNIC_Value)
+		if err != nil {
+			klog.Warning("failed to list ovn-ic lr policy ", err)
+			return
+		}
+		for _, lrPolicy := range lrPolicyList {
+			err := c.ovnClient.DeleteRouterPolicy(lr, lrPolicy.UUID)
+			if err != nil {
+				klog.Warningf("deleting router policy failed %v", err)
+			}
+		}
+		return
+	}
+
+	policyMap := map[string]string{}
+	lrPolicyList, err := c.ovnClient.GetLogicalRouterPoliciesByOptions(util.OVNIC_Key, util.OVNIC_Value)
+	if err != nil {
+		klog.Warning("failed to list ovn-ic lr policy ", err)
```

Comment:
all the warning should be replaced with Errorf

Reply:
done